_docs/_latest/develop/dev/setup.md

```diff
@@ -242,6 +242,23 @@ cd <virtual environment>/lib/python2.7/site-packages/google/cloud
 ln -s <path to your git source code>/google/cloud/forseti forseti
 ```
 
+## Setting up instrumentation
```

Comment:
Did we also document somewhere, the new API that needs to be enabled?

Reply:
Documented in `using-instrumentation.md`.